src/OrchardCore.Modules/OrchardCore.Workflows/Services/SecurityTokenService.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Localization;
+using Newtonsoft.Json;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Workflows.Services
+{
+    public class SecurityTokenService : ISecurityTokenService
+    {
+        private readonly ITimeLimitedDataProtector _dataProtector;
+        private readonly IClock _clock;
+
+        public SecurityTokenService(
+            IDataProtectionProvider dataProtectionProvider, 
+            IClock clock,
+            IStringLocalizer<SecurityTokenService> localizer)
+        {
+            _dataProtector = dataProtectionProvider.CreateProtector("Tokens").ToTimeLimitedDataProtector();
```

Comment:
Note: now that you merged the `dev` branch into your branch, you'll probably want to add a module dependency on `OrchardCore.DataProtection` to ensure the `IDataProtectionProvider` instance you get is tenant-isolated (and not provided by the host).

Reply:
I assume you get an instance from an indirect reference to a module that requires it, but you require it too and you should declare a dependency on it.